services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/kafka/RecordHandler.java

```diff
@@ -349,7 +357,7 @@ private void installTransitFlow(final InstallTransitFlow command) throws SwitchO
     private void doProcessOneSwitchFlow(final CommandMessage message, String replyToTopic, Destination replyDestination)
             throws FlowCommandException {
         InstallOneSwitchFlow command = (InstallOneSwitchFlow) message.getData();
-        logger.debug("creating a flow through one switch: {}", command);
+        logger.info("Creating a flow through one switch: {}", command);
```

Comment:
`one switch flow` is more suitable that `flow through one switch`. Do not log whole command here.